.gitlab/docker/Dockerfile.spack_build

```
@@ -6,6 +6,8 @@ ARG BUILD_DIR
 
 ARG CMAKE_COMMON_FLAGS
 ARG CMAKE_FLAGS
+# Overridden automatically when using .container-builder
+ARG NUM_PROCS=16
```

Should we leave this as
```suggestion
ARG NUM_PROCS
```
so that we notice if it's for some reason not defined anymore? I think I'd rather have this be an error than silently fall back to 16 if the internals of `.container-builder` change for some reason.